dev/Prototypes/ItemsViewPrototype/FileExplorerScenario.xaml

```diff
@@ -72,7 +78,10 @@
                 <c:RecyclingElementFactory RecyclePool="{StaticResource recyclePool}">
                     <DataTemplate x:Key="IconItemTemplate" x:DataType="dep:Item">
                         <l:ItemContainer Layout="{StaticResource iconItemLayout}" Background="LightGray" BorderBrush="Gray" BorderThickness="1">
-                            <Image Grid.Column="0" Width="60" Height="60">
+                            <l:DragAndDropBehavior.Source>
```

Comment:
Perhaps we need to bake this behavior into ItemContainer instead of setting it in the app.

Reply:
Yes. As it stands now, it seems like we need to hook this up to each and every ItemContainer we create. Which means each ViewDefinition needs this setup.